shared/data/etags.json

```diff
@@ -1 +1 @@
-{"config-etag":"W/\"09fa14b9e65122d55e9f024922140627\""}
\ No newline at end of file
+{"config-etag":"W/\"32052367-40070-2022-05-27T19:47:39.127Z\""}
```

Comment:
This doesn't look right

Reply:
You're right it doesn't!? this is pulled from the remote resource headers... maybe there's a bug in that code as this is a timestamp along with other things.

Reply:
Oh wait it could be an artefact of me serving it locally and bundling from localhost!